lib/WasmReader/WasmParseTree.h

```diff
@@ -15,8 +15,10 @@ namespace Wasm
             Void,
 #define WASM_LOCALTYPE(token, name) token,
 #include "WasmKeywords.h"
-            Limit
+            Limit,
+            Unreachable
```

Comment:
I guess I see why you did this way, but in a lot of cases Limit == count of the enum, and there is risk of someone misusing like that. Might be good to rename to avoid confusion.